pkg/chains/formats/intotoite6/taskrun/taskrun.go

```diff
@@ -92,6 +94,18 @@ func materials(tro *objects.TaskRunObject) []slsa.ProvenanceMaterial {
 		return mats
 	}
 
+	// Retrieve structured provenance for inputs.
+	ssts := artifacts.ExtractStructuredTargetFromResults(tro, artifacts.ArtifactsInputsResultName, logger)
+	for _, s := range ssts {
+		splits := strings.Split(s.Digest, ":")
+		alg := splits[0]
+		digest := splits[1]
+		mats = append(mats, slsa.ProvenanceMaterial{
+			URI:    s.URI,
+			Digest: map[string]string{alg: digest},
+		})
+	}
+
```

Comment:
We also need to do this for [pipelinerun level material extraction](https://github.com/tektoncd/chains/blob/e0f5af8998ce80a477af7fd1fc25fed9147f5fc7/pkg/chains/formats/intotoite6/pipelinerun/pipelinerun.go#L171-L217).

Reply:
Good catch thanks! Added here. The tests need to be added later as there is an update needed from Tekton Pipeline side for json marshall for object results. I've already talked to Yongxuan about it. Added a TODO for that as well